TICC.ino

```diff
@@ -61,37 +41,23 @@ const int CALIBRATION2 = 0x1C;            // default 0x00_0000
 // max clock speed: 20 mHz
 // Using TDC7200 timing mode 1...
 
-// global vars
-long long unsigned stop_time_A;
-long long unsigned stop_time_B;
-long result_A;
-long result_B;
-int CSB;
-int ENABLE;
-
-  int timeresult_A = 0;
-  int clockresult_A = 0;
-  int calresult_A = 0;
-  int timeresult_B = 0;
-  int clockresult_B = 0;
-  int calresult_B = 0;
 
 void setup() {
   // start the SPI library:
   SPI.begin();
   
-  pinMode(int_pin, INPUT);
+  pinMode(intPin, INPUT);
   
-  pinMode(ENABLE_A,OUTPUT);
-  pinMode(INTB_A,INPUT);
-  pinMode(CSB_A,OUTPUT);
-  pinMode(STOP_A,INPUT);
-  pinMode(ENABLE_B,OUTPUT);
-  pinMode(INTB_B,INPUT);
-  pinMode(CSB_B,OUTPUT);
-  pinMode(STOP_B,INPUT);
+  pinMode(ChA->ENABLE,OUTPUT);
+  pinMode(ChA->INTB,INPUT);
+  pinMode(ChA->CSB,OUTPUT);
+  pinMode(ChA->STOP,INPUT);
+  pinMode(ChB->ENABLE,OUTPUT);
+  pinMode(ChB->INTB,INPUT);
+  pinMode(ChB->CSB,OUTPUT);
+  pinMode(ChB->STOP,INPUT);
   
-  attachInterrupt(interrupt, coarse_timer, RISING);
+  attachInterrupt(interrupt, coarseTimer, RISING);
   
   TDC_setup(ChA);
   TDC_setup(ChB);
```

Comment:
These just become
```
TDC_setup(&ChanA)
TDC_setup(&ChanB)
```

And it becomes clear that you're just sending pointers to the static structs used in here.